### ISSUE TITLE
Issue with YLYK01YL and latest update

### ISSUE THREAD

**Issue by chertvl** [custom-components/ble_monitor#377]
Hello.
I have a some problems with running YLYK01YL in latest 2.6.2 version of BLE Monitor.
When I just updated my remotes stops working, sensors are 'unknown', error "no encryption key v2/v3 found" in logs every time I press any button.
Rolled back to 2.4.3 for now, and now all works perfectly. (Yes im sure in correct of my encryption key, thanks).
Can you fix it please?

Thanks you for this integration!

**Comment by Ernst79** [comment]
Hi @chertvl 

I can't reproduce the issue, so I need a little help from you to solve it. Could you do me a favor and install 2.6.2, and modify the following part on line 590 in `ble_monitor/ble_parser/xiaomi.py`. I've added the two _LOGGER lines. After a restart, press a key and look in the Home Assistant log. Copy the two lines. 

If you don't want to post your encryption key, change 2 characters, but change them exactly the same in both errors in the log. Make sure you also keep it capitalized in both errors, if it is in capitals. This is important, because I suspect some issue with lower case and capitalized letters. 

```
def decrypt_mibeacon_legacy(self, data):
    _LOGGER.error("mac %s", self.xiaomi_mac.lower())
    _LOGGER.error("keylist %s", self.aeskeys)
    try:
        # check for minimum length of encrypted advertisement
        if self.xdata_length < 9:
```

**Comment by chertvl** [comment]
Decryption MiBeacon V2/V3 advertisement failed: No encryption key found
20:00:54 – (ОШИБКА) Passive BLE monitor (кастомная интеграция) - первое сообщение получено 20:00:28 и повторялось 8 раз
keylist {b'\xf8$A\xe3\xc0\x86': b'\xc6\xbb\xf4\xc6{\x1c\xb8\x91\xda\x13\xeea'}
20:00:54 – (ОШИБКА) Passive BLE monitor (кастомная интеграция) - первое сообщение получено 20:00:28 и повторялось 8 раз
mac b'\xf8$a\xe3\xc0\x86'
20:00:54 – (ОШИБКА) Passive BLE monitor (кастомная интеграция) - первое сообщение получено 20:00:28 и повторялось 8 раз


That what I have at the Logs. All in the true state, I have not made changes in text. Thanks.

**Comment by Ernst79** [comment]
Ok, thanks, I see the issue, it's indeed something with capitals and lower case letters. I'll figure out where the error is and come back with an update.

**Comment by Ernst79** [comment]
Should be fixed with 2.6.3. Apparently my MAC translates to lowercase in bytes, so I didn't find the error while testing. It was applying a lowercase to the byte characters.

**Comment by chertvl** [comment]
Can confirm, at 2.6.3 bug is no longer there, thank you very much
